tasks/css.js

```diff
@@ -139,6 +139,11 @@ gulp.task('css:bundle', () => {
         .pipe(gulpif(config.isDev,
             sourcemaps.write('.')
         ))
+
+        // If the package version name is set to `true`, version the css file name with the package number.
+        .pipe(gulpif(config.css.usePackageVersion,
```

Comment:
If you do the rename here before the sourcemap write above, will the sourcemaps still link properly?  (worth checking the name of the file linked in the CSS file produced – may work this out automatically)

Reply:
Good spot. I checked this and got the below so looks ok to me? I did fix another issue that I spotted while checking this though, the suffix for the `.min` wasn't being applied (from a change I made) I've added a fix for this now.

![screen shot 2018-01-25 at 11 58 48 am](https://user-images.githubusercontent.com/2299779/35387253-d73b6c24-01c6-11e8-83c1-be1f19c061e3.png)